cmd/psql/psql.go

```diff
@@ -0,0 +1,60 @@
+package psql
+
+import (
+	"errors"
+
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/libs/lakebase"
+	"github.com/spf13/cobra"
+)
+
+func New() *cobra.Command {
+	return newLakebaseConnectCommand()
+}
+
+func newLakebaseConnectCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "psql [DATABASE_INSTANCE_NAME] [-- PSQL_ARGS...]",
+		Short:   "Connect to the specified Database Instance",
+		Args:    cobra.MinimumNArgs(0),
+		GroupID: "database",
+		Long: `Connect to the specified Database Instance.
+
+This command requires a psql client to be installed on your machine for the connection to work.
+
+You can pass additional arguments to psql after a double-dash (--):
+  databricks psql my-database -- -c "SELECT * FROM my_table"
+  databricks psql my-database -- --echo-all -d "my-db"
+`,
+	}
+
+	// Wrapper for [root.MustWorkspaceClient] that disables loading authentication configuration from a bundle.
+	mustWorkspaceClient := func(cmd *cobra.Command, args []string) error {
```

Comment:
Why do we not want to use bundle authentication? For most other commands we use bundle auth so this is inconsistent.

Reply:
My understanding was that the command that runs outside a bundle (like `databricks psql`) should use this (I copied this from the `sync` command that is run outside the bundle). Is that understanding incorrect?

Reply:
I ended up using bundle authentication with this command